CMakeLists_files.cmake

```diff
@@ -147,6 +147,7 @@ list (APPEND PUBLIC_HEADER_FILES
   opm/autodiff/AutoDiff.hpp
   opm/autodiff/BackupRestore.hpp
   opm/autodiff/BlackoilDetails.hpp
+	opm/autodiff/BlackoilLegacyDetails.hpp
```

Comment:
yeah, fixed. thanks.

(maybe `indent-tabs-mode: nil` should be added to the modelines of `*.cmake` files.)